aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -37,30 +37,102 @@ Referencing ASP.NET Core in this manner is only supported for projects targeting
 
 ## Include Blazor extensibility
 
-Blazor supports WebAssembly (WASM) and Server [hosting models](xref:blazor/hosting-models). Unless there's a specific reason not to, a [Razor components](xref:blazor/components/index) library should support both hosting models. A Razor components library must use the [Microsoft.NET.Sdk.Razor SDK](xref:razor-pages/sdk).
+Blazor supports WebAssembly (WASM) and server-based [hosting models](xref:blazor/hosting-models). Unless there's a specific reason not to support both hosting models, a [Razor components](xref:blazor/components/index) library should support both hosting models. A Razor components library must use the [Microsoft.NET.Sdk.Razor SDK](xref:razor-pages/sdk).
 
 ### Support both hosting models
 
-To support Razor component consumption from both [Blazor Server](xref:blazor/hosting-models#blazor-server) and [Blazor WASM](xref:blazor/hosting-models#blazor-webassembly) projects, use the following instructions for your editor.
+To support Razor component consumption from both [Blazor Server](xref:blazor/hosting-models#blazor-server) and [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) projects, use the following instructions for your editor.
 
 # [Visual Studio](#tab/visual-studio)
 
-Use the **Razor Class Library** project template. The template's **Support pages and views** checkbox should be deselected.
+Use the **Razor Class Library** project template. Disable the template's **Support pages and views** check box.
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
 Run the following command in the integrated terminal:
 
 ```dotnetcli
-dotnet new razorclasslib
+dotnet new razorclasslib --support-pages-and-views false
 ```
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-Use the **Razor Class Library** project template.
+Use the **Razor Class Library** project template. Disable the template's **Support pages and views** check box.
 
 ---
 
+::: moniker range=">= aspnetcore-5.0"
+
+The library generated from the project template:
+
+* Targets .NET 5 with the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks) of `net5.0`.
```

Comment:
Do you think we need to specify net5? It'll quickly become out of date. How about "Targets the current .NET framework based on the install SDK. For instance if you .NET 5.0 installed, this would target .NET 5.